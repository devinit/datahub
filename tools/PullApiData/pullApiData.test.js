import prettyFormat from 'pretty-format';
import COUNTRIES_QUERY from '../../private/graphql/Countries.graphql';
import {getData} from '.';

describe('pull api data tests', () => {
  it('should return countries used in country search input', async () => {
    const data = await getData(COUNTRIES_QUERY);
    expect(data.countries.length).toBeGreaterThan(2);
    expect(prettyFormat(data.countries[0])).toMatchSnapshot();
  }, 20000);
});
