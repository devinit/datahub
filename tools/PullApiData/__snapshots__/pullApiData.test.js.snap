// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pull api data tests should return countries used in country search input 1`] = `
"Object {
  \\"id\\": \\"afghanistan\\",
  \\"name\\": \\"Afghanistan\\",
}"
`;
